Userland/Libraries/LibJS/AST.cpp

```diff
@@ -99,10 +99,25 @@ Value FunctionDeclaration::execute(Interpreter& interpreter, GlobalObject&) cons
     return {};
 }
 
+// 15.2.5 Runtime Semantics: InstantiateOrdinaryFunctionExpression, https://tc39.es/ecma262/#sec-runtime-semantics-instantiateordinaryfunctionexpression
 Value FunctionExpression::execute(Interpreter& interpreter, GlobalObject& global_object) const
 {
     InterpreterNodeScope node_scope { interpreter, *this };
-    return OrdinaryFunctionObject::create(global_object, name(), body(), parameters(), function_length(), interpreter.lexical_environment(), kind(), is_strict_mode() || interpreter.vm().in_strict_mode(), is_arrow_function());
+    Environment* func_env = interpreter.lexical_environment();
```

Comment:
Please use `auto` in most cases.

```suggestion
    auto* func_env = interpreter.lexical_environment();
```